mysql-test/mroonga/storage/collation/utf8mb4_uca1400_ai_ci/r/french.result

```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS diaries;
+SET NAMES utf8mb4;
+CREATE TABLE diaries (
+content varchar(256) COLLATE utf8mb4_uca1400_ai_ci,
+FULLTEXT INDEX (content)
+) DEFAULT CHARSET=utf8mb4;
+INSERT INTO diaries VALUES ("Je suis un garçon.");
+INSERT INTO diaries VALUES ("noçrag nu sius eJ");
```

Comment:
Can we use a valid sentence instead of a reversed text?

Reply:
fix: [96429e8](https://github.com/mroonga/mroonga/pull/908/commits/96429e8eff8965b25ec43db81e721bad718c3241) Sure.